lib/auth/auth.go

```diff
@@ -119,6 +119,14 @@ const (
 	mfaDeviceNameMaxLen = 30
 )
 
+const (
+	OSSDesktopsCheckPeriod  = 5 * time.Minute
+	OSSDesktopsAlertID      = "oss-desktops"
+	OSSDesktopsAlertMessage = "Teleport Community Edition only allows 3 Non-AD desktops configured. " +
+		"You won't be able to connect to any of them if there are more" +
+		"Please contact Sales to upgrade your license"
```

Comment:
Not sure about this copy - if they are on OSS then they don't have any license to upgrade. Maybe we should just say "contact us to upgrade"

Reply:
@xinding33 could you advise on message here? And also on what link we should provide for the user here